### PR TITLE
Connect Wallet on Home Screen

### PR DESCRIPTION
**Pull request by youngkidwarrior**

### CODE REVIEW

**Comment by youngkidwarrior** [comment]
* **#307** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/307?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `dev`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @youngkidwarrior and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/app/features/home/screen.tsx [hunk 1]
```diff
@@ -117,45 +145,53 @@ export function HomeScreen() {
...
+          <OpenConnectModalWrapper h={'$6'} width={'100%'}>
+            <Button
+              px={'$3.5'}
+              h={'$6'}
+              width={'100%'}
+              theme="accent"
+              borderRadius={'$4'}
+              onPress={() => {
+                // @todo onramp / deposit
+                toast.show('Coming Soon: Deposit')
```
Comment (0xBigBoss) [inline packages/app/features/home/screen.tsx:157]: does it actually work? what's going on with this toast?

Comment (youngkidwarrior) [reply packages/app/features/home/screen.tsx:158]: this should work, now. Oddly `stopPropagation()` still fires the `onPress()` event below it

**Comment by youngkidwarrior** [comment]
@0xBigBoss I added a few fixes outside this PR scope just to get them to prod faster

File: packages/app/features/home/screen.tsx [hunk 2]
```diff
@@ -70,23 +73,48 @@ export function HomeScreen() {
...
+                                  fontFamily={'$mono'}
+                                  fontSize={'$15'}
+                                  lineHeight={'$14'}
+                                  fontWeight={'500'}
+                                  zIndex={1}
+                                >
+                                  N/A
+                                </Paragraph>
+                              )
+                            case totalBalance === undefined:
```
Comment (0xBigBoss) [inline packages/app/features/home/screen.tsx:91]: ideally, we'd make sure there was no error when fetching too or use the react query isLoading.

**Review by 0xBigBoss — APPROVED** [review]

**Pull request merged**